backend/dissemination/api/api_v1_1_0/create_schema.sql

```diff
@@ -34,13 +34,13 @@ $$
 ;
 
 -- This is the description
-COMMENT ON SCHEMA api_v1_0_3 IS
+COMMENT ON SCHEMA api_v1_1_0 IS
     'The FAC dissemation API version 1.0.3.'
 ;
 
 -- https://postgrest.org/en/stable/references/api/openapi.html
 -- This is the title
-COMMENT ON SCHEMA api_v1_0_3 IS
+COMMENT ON SCHEMA api_v1_1_0 IS
 $$v1.0.3
```

Comment:
Lets get rid of this for now. Just make this weirdness about the version number go away completely.

Reply:
Removed `$$v1.0.3` from create_schema.sql